x/ethbridge/client/cli/tx.go

```diff
@@ -20,46 +20,49 @@ import (
 //nolint:lll
 func GetCmdCreateEthBridgeClaim(cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
-		Use:   "create-claim [ethereum-chain-id] [bridge-contract] [nonce] [symbol] [token-contract] [ethereum-sender-address] [cosmos-receiver-address] [validator-address] [amount] [claim-type]",
+		Use:   "create-claim [bridge-contract] [nonce] [symbol] [ethereum-sender-address] [cosmos-receiver-address] [validator-address] [amount] [claim-type] --ethereum-chain-id [ethereum-chain-id] --token-contract-address [token-contract-address]",
```

Comment:
think `[bridge-contract]` is now `[bridge-registry-contract]` will probably have a merge conflict w master. Wanted the names to be consistent across sides, i remember being confused previously about which contract address I should actually be adding to the different relayer commands. Think when you point it to the registry instead it got worked out but wanted to be explicit anyway.

Reply:
https://github.com/cosmos/peggy/pull/175/commits/39fc381bbc67e0464e69b2979197b37d108e118d

Reply:
nice